backend/api/Services/IsarService.cs

```diff
@@ -71,22 +64,14 @@ public IsarService(ILogger<IsarService> logger, IDownstreamWebApi downstreamWebA
             return response;
         }
 
-        public IsarMissionDefinition GetIsarMissionDefinition(Mission mission)
-        {
-            var tasks = mission.PlannedTasks.Select(task => new IsarTaskDefinition(task, mission));
-            return new IsarMissionDefinition(tasks: tasks.ToList());
-        }
-
-        public async Task<IsarServiceStartMissionResponse> StartMission(
-            Robot robot,
-            IsarMissionDefinition missionDefinition
-        )
+        public async Task<IsarMission> StartMission(Robot robot, Mission mission)
         {
+            var missionDefinitionObject = new IsarMissionDefinition(mission);
```

Comment:
Would it be better to just do this directly at line 74?

Reply:
Fixed